WcaOnRails/app/models/competition.rb

```diff
@@ -650,7 +650,7 @@ def city_and_country
 
   def result_cache_key(view)
     results_updated_at = results.order('updated_at desc').limit(1).pluck(:updated_at).first
-    [id, view, results_updated_at.try(:iso8601) || ""]
+    [id, view, results_updated_at.try(:iso8601) || "", I18n.locale]
```

Comment:
Just curious, does the order of the elements of this array matter? I'm guessing no.